jboss-kie-kieserver/added/launch/jboss-kie-kieserver.sh

```diff
@@ -36,6 +36,15 @@ function prepareEnv() {
     unset KIE_SERVER_KAFKA_EXT_GROUP_ID
     unset KIE_SERVER_KAFKA_EXT_MAX_BLOCK_MS
     unset KIE_SERVER_KAFKA_EXT_TOPICS
+    unset KIE_SERVER_KAFKA_JBPM_EVENT_EMITTER_ACKS
```

Comment:
please organize the envs on alphabetical order.

Reply:
ok